guides/common/modules/con_best-practices-for-deploying-smartproxies.adoc

```diff
@@ -0,0 +1,18 @@
+[id="best-practices-for-deploying-{smart-proxy-context}_{context}"]
+= Best practices for deploying {SmartProxies}
+
+* Install a {SmartProxyServer} in each of your geographic locations.
+By assigning a {SmartProxy} to each location, you decrease the load on {ProjectServer}, increase redundancy, and reduce bandwidth usage.
++
+[NOTE]
+====
+The maximum number of {SmartProxyServers} that {ProjectServer} can support has no fixed limit.
+It was tested that a {ProjectServer} can support 17 {SmartProxyServers} with 2 vCPUs.
+====
+* Your {ProjectServer} contains an integrated {SmartProxyServer}.
```

Comment:
```suggestion
* Your {ProjectServer} contains an integrated {SmartProxy}.
```

I do not like the word "integrated" at all. However, I cannot provide a good/better alternative, so I'm OK.